### PR TITLE
fix: Statistics suffix fontsize inherit from value style

### LINKED ISSUE #28216: Statistic组件升级4.9.1后出现字体大小问题，是否可以自定义prefix、suffix样式
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

image
image

### What problem does this feature solve?
Statistic处理对于单纯字符前后缀样式控制

### What does the proposed API look like?
prefixStyle、suffixStyle

### PR DESCRIPTION
**Pull request by rdmclin2**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

resolve https://github.com/ant-design/ant-design/issues/28216

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fix Statistics decimal font size. |
| 🇨🇳 Chinese | 修复 Statistics 小数点字体大小问题。 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by rdmclin2** [comment]
close #28216

File: components/statistic/style/index.less [hunk 1]
```diff
@@ -20,9 +20,6 @@
     &-value {
       display: inline-block;
       direction: ltr;
-      &-decimal {
```
Comment (zombieJ) [inline components/statistic/style/index.less:23]: 是不是应该提上来？这样直接丢样式了。

File: components/form/index.en-US.md [hunk 2]
```diff
@@ -415,6 +415,11 @@ Your name path should also contain Form.List `name`:
 
 dependencies should be `['users', 0, 'name']`
 
+
+### Why `normalize` do not support async?
+
+React can not get correct interaction of controlled component with async value update. When user trigger `onChange`, component will do no response since `value` update is async. If you want to trigger value update async, you should use customize component to handle value state internal and pass sync value control to Form instead.
```
Comment (afc163) [inline components/form/index.en-US.md:421]: 包含了不相关的 changes

**Comment by afc163** [comment]
@rdmclin2

File: components/statistic/style/index.less (hunk 1 above)
Comment (rdmclin2) [reply components/statistic/style/index.less:23]: 是decimal 继承 value 的字体样式。

**Pull request merged**